src/actions/apiActions.js

```diff
@@ -197,17 +197,21 @@ export const formatCourse = (formCourse, type) =>{
     };
 }
 
-const parseTime = (time) =>{
+const courseName = (form, type) => {
+    if(type === "T"){
+        return "1:1 " + form["Instructor"].value + form[""]
+    }
+}
+
+export const parseTime = (time) =>{
     let formattedTime;
     if(typeof time === "string"){
-        if(time.indexOf("AM") > -1 || time.indexOf("PM") > -1){
-            formattedTime = new Date();
-            formattedTime.setHours(Number(time.substring(0,time.indexOf(":"))));
-            formattedTime.setMinutes(Number(time.substring(time.indexOf(":")+1,time.indexOf(" "))));
-            formattedTime.setSeconds(0);
-        }
+        let Hour = time.substr(17, 2);
+        let to12HourTime = (Hour % 12) || 12;
+        let ampm = Hour < 12 ? " am" : " pm";
+        formattedTime = to12HourTime + time.substr(19, 3) + ampm;
```

Comment:
Suggest using the [Date object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date)'s methods for getting hours of the time instead of parsing a string, especially since you seem to be converting the created Data objects to strings before calling this function.